api-client/src/main/java/com/xing/api/ContentRange.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2016 XING AG (http://xing.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.xing.api;
+
+import java.io.Serializable;
+
+/**
+ * Represents the content range data of the response. This is useful when a paginated response is being accepted.
+ * Contains information about the current content offset, range and total available values.
+ * <p>
+ * XWS returns a <strong>Xing-Content-Range</strong> header for all paginated requests. The format is as follows:
+ * <ul>
+ * <li><i>"Xing-Content-Range: items [offset]-[last]/[total]"</i> - if the total number is <strong>known</strong>.</li>
+ * <li><i>"Xing-Content-Range: items [offset]-[last]/*"</i> - if the total number is <strong>unknown</strong>.</li>
+ * <li><i>"Xing-Content-Range: items &#42;/0"</i> - if the response returned an empty collection.</li>
+ * </ul>
+ */
+public final class ContentRange implements Serializable {
+    private static final long serialVersionUID = 1L;
+    private static final int MINIMUM_LENGTH = 9; // Assumes the length of minimum possible value "items */0".
+
+    static final String HEADER_NAME = "Xing-Content-Range";
+
+    static ContentRange parse(String headerValue) {
```

Comment:
Have you considered using Regex with groups to parse the string. This would prevent some method calls and be shorter. Especially since the pattern is always the same - only the values differ.